LICENSE

```diff
@@ -11,4 +13,3 @@ ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
 WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
 ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
 OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
-
```

Comment:
I did leave a newline at the end of the file (there were two). I suspect this is a github rendering issue. In fact, if I attempt to edit [the file in my branch](https://github.com/waldyrious/wego/edit/patch-1/LICENSE), it does show up in the editor with 16 lines, the last one being a newline. Can you confirm?